infra/resources/modules/commons/apim.tf

```diff
@@ -246,3 +246,33 @@ resource "azurerm_api_management_subscription" "io_backend" {
   state               = "active"
   allow_tracing       = false
 }
+
+####################################################################################
+# Support User
+####################################################################################
+resource "azurerm_api_management_user" "support_user" {
```

Comment:
**nitpick**: I'd not add the `user` suffix because it's pretty much clear what type of resource is: `azurerm_api_management_user`. I find it redundant

```suggestion
resource "azurerm_api_management_user" "support" {
```

Reply:
should this change be applied for the other resources too? Specifically `azurerm_api_management_group_user` and `azurerm_api_management_subscription`.
or maybe we could use a more explicit name?

Reply:
Resolved in eda5c7e8c8214cd1ce743bd5145934c78ab591bf